src/components/organisms/AdminVenueView/components/RunTabView/RunTabView.tsx

```diff
@@ -39,7 +39,7 @@ export const RunTabView: React.FC<RunTabViewProps> = ({ venue }) => {
         </div>
       </AdminSidebar>
       <AdminShowcase className="RunTabView__main" variant="no-scroll">
-        <div className="RunTabView__toolbar RunTabView--spacing">
+        <div className="RunTabView__toolbar RunTabView--spacing mod--hidden">
```

Comment:
This should make it render only to people who have the flag set on their user.

```suggestion
        <TesterRestricted>
        <div className="RunTabView__toolbar RunTabView--spacing">
```

Alternatively, the "soft removal" (you can copy the approach from `disableSettings.ts`) is:

```suggestion
      {!DISABLED_DUE_TO_1583 &&
        <div className="RunTabView__toolbar RunTabView--spacing mod--hidden">
```